### ISSUE TITLE
Unable to assign an assignment to oneself

### ISSUE THREAD

**Issue by petrzpav** [InternetGuru/cad#7]
E.g. following error message

```
Updating user repository for lukasbarinka ... {"message":{"access_level":["should be greater than or equal to Owner inherited membership from group b201"]}} EXCEPTION: Invalid request https://gitlab.com/api/v4/projects/23617910/members [400]
```

**Comment by jiripavelka** [comment]
Looks like it wouldn't even make sense to set a developer rights to a project owner. Solution may be either to ignore this kind of error or (better) to somehow detect the same user and not even try to set rights.

**Comment by petrzpav** [comment]
Fixed in 1a5a3f6
